### ISSUE TITLE
RP Meeting - 4/5

### ISSUE THREAD

**Issue by sarahcleary** [APSIMInitiative/ReferencePanel#85]
**Grape Vine Model Review**
9:30-10:30 AEST

@junqi108 to present to the RP

Questions:
@HamishBrownPFR - will this be sufficient time?  do we need to extend this meeting?
@APSIMInitiative/reference-panel - who else needs to be invited to this review? 

@junqi108; @HamishBrownPFR - any instructions for the @APSIMInitiative/reference-panel prior to this meeting?  Anything they should do to prepare?

**Comment by sarahcleary** [comment]
# 2021-05 Minutes
### Attendees:  
@MarkLieffering; @LouisAK; @hol353; ;  @peter-devoil;  @yashvirchauhan;  @sarahcleary; @jbrider;  @HamishBrownPFR;  @hut104; @sarchontoulis;  @sno036  @junqi108; @Keith-Pembleton  @EnliWang; 
### Apologies/Not in attendance: 
@kchenu;  @JulianneLilley; 
### Tuesday 4th May 2021 - 9:30 am AEST
Meeting recorded – which can be accessed on Teams link for meeting attendees. 
Review of Grape Vine Model -https://github.com/APSIMInitiative/ApsimX/issues/1822  
@junqi108 ran through the powerpoint – Introduction of APSIM Grapevine model  - 
[Introduction of APSIM grapevine model.pdf](https://github.com/APSIMInitiative/ReferencePanel/files/6424830/Introduction.of.APSIM.grapevine.model.pdf)


## Discussion with group on model 
Recording from 52min to 1:20 min.

## Work required for Model to be accepted into Release
** Preamble require a bit more information with cautions/caveats for new users
** Tidy up documentation based on Neil's comments
** Run the entire model for a 20 year period to ensure stability
** Undertake Sensibility tests for water stress responses and any other factors without validation data
** Check integration works by simulating sheep in the vineyard. Ensure the sheep each leaves not roots.
** Put irrigation into simulations

## Debrief discussion from 1:20-end
**Key Points for Future Review Meetings** 
Presentation and Autodocumentation to be circulated a week prior to the review meeting
Model to be moved to “under review” folder at the same time as the documentation being circulated to the RP
RP members to provide any major feedback prior to the meeting.  

**Outcomes from meeting:**
A list of agreed points requiring work.  If these points are fixed/completed, then the model will be ready for release

**Comments on new process:**
Increase visibility with the RP on new models
Use Junqi’s powerpoint as template for future presentations 
As per previous discussions, if validation data isn’t available, will accept sensibility analysis

**Comment by sarahcleary** [comment]
Note additional comments on https://github.com/APSIMInitiative/ApsimX/issues/1822#

**Comment by sno036** [comment]
@sarahcleary, @HamishBrownPFR and I were chatting ... this review worked because Junqi was presenting and fielding questions while Hamish was taking notes and moderating. This was more of a happy accident but something we should make sure happens consistently for future reviews. Particularly so when the presenter is not one of the RP members probably.

**Comment by sarahcleary** [comment]
I agree, the meeting went very smoothly.  I don't believe this was by accident and noted that @HamishBrownPFR had worked closely with @junqi108 in preparation for this meeting.  I also understand @hut104 had provided @junqi108 with feedback prior to the meeting.  It might be a good idea for a RP rep (or someone close to the RP) reviews and/or assists with preparation for the meeting.

**Comment by hol353** [comment]
I agree it went well. My only comment is that the presentation went a little long. Would be good to suggest that presenters have 30 minutes. What do you think?

**Comment by sno036** [comment]
My two-cents on duration is that we need to be flexible. For the grape model I thought the issue was more that we scheduled too short a time slot than that the presentation itself was too long. They had a lot to cover (the new geometry as well as multi-year phenology and then the model and data). Other presentations may well be much simpler with fewer concepts to introduce and may well only need 30 min. If the 'oral examination' method is going to be effective then we do have to allow time for discussion and questions and then the list of items to be completed before release.

**Comment by peter-devoil** [comment]
A shorter time for pedestrian subjects (eg another annual field crop) could be possible. But this exercise had several major new pieces of functionality which were only briefly discussed. I'd leave any decision on amount of time required to the facilitators

**Comment by hol353** [comment]
Yep ok. My real issue was the too short a time slot and someone else put a meeting into my calendar immediately following the RP meeting. I missed the important discussion. I'll talk to Neil and get an update.

**Comment by sarahcleary** [comment]
@hol353 - the recording is on the teams link - let me know if you can't access it.  Main discussion from 52 min point to 1:20 if you would like to listen to it

**Comment by sarahcleary** [comment]
@JulianneLilley, @kchenu - recording is available if you would like to listen. Any issues accessing it, please give me a call

**Comment by hol353** [comment]
@sarahcleary I can't see a recording of the previous meeting.

**Comment by sarahcleary** [comment]
@hol353 - sent by email
